### ISSUE TITLE
No way to select text behind a hyperlink?

### ISSUE THREAD

**Issue by Earnestly** [The-Compiler/qutebrowser#1079]
Sometimes it's useful to select text and copy it elsewhere.  If this text happens to be guarded by a hyperlink there is no way afaict to select this text without activating the link.

In firefox this was accomplished by holding `alt`, this would allow you to select the text without triggering the link itself.

Sorry if there is a way or a duplicate bug report, I have tried to look but could have missed it.

**Comment by The-Compiler** [comment]
I guess you mean selecting a part of a hyperlink text?

I don't think there's a way for me to do this via the Python/C++ API - it might be possible to do this via JS however (or at least I see Chromium extensions for it), so I'll keep it open for now.

**Comment by damarusama** [comment]
Yes I am having the same trouble, for example when I want to copy an email, but it's embeded in a url, I can't simply copy the text (and paste in in mutt for example). There would ned to be another item in the right click menu that adds Copy text, under Copy link address.

**Comment by jmburgos** [comment]
I also noticed this.  Sometimes I need to copy a bit of text from a website, but if that text has an hyperlink I can only copy the link address, no the text itself.

**Comment by The-Compiler** [comment]
FWIW you can always use caret mode (e.g. search for the text and then press `v`) to select a link text or even part of it.
